docs/content/preview/explore/going-beyond-sql/follower-reads-ysql.md

```diff
@@ -33,15 +33,19 @@ For an illustration of the performance improvements using leader leases, see [Lo
 
 ## Configuration parameters
 
-Two YSQL configuration parameters control the behavior of follower reads:
+The following YSQL configuration parameters control the behavior of follower reads:
 
 - `yb_read_from_followers` controls whether or not reading from followers is enabled. The default value is false.
 
 - `yb_follower_read_staleness_ms` sets the maximum allowable staleness. The default value is 30000 (30 seconds).
 
   Although the default is recommended, you can set the staleness to a shorter value. The tradeoff is the shorter the staleness, the more likely some reads may be redirected to the leader if the follower isn't sufficiently caught up. You shouldn't set `yb_follower_read_staleness_ms` to less than 2x the [raft_heartbeat_interval_ms](../../../reference/configuration/yb-tserver/#raft-heartbeat-interval-ms) (which by default is 500 ms).
 
-Note that even if the tablet leader is on the closest node, you would still read from `Now()-yb_follower_read_staleness_ms`. Therefore, when follower reads are used, the read is always stale, even if you are reading from a tablet leader.
+  Note that even if the tablet leader is on the closest node, you would still read from `Now()-yb_follower_read_staleness_ms`. Therefore, when follower reads are used, the read is always stale, even if you are reading from a tablet leader.
+
+  If the users want to monitor whether any reads are getting redirected to the leader, they can monitor the metric - `consistent_prefix_failed_reads`, it will show the number of such requests redirected.
```

Comment:
s/redirected/redirected from the current node/ ? 
(Essentially this metric "on the follower/stale-node" is what we care about.)